addOns/bruteforce/src/main/java/com/sittinglittleduck/DirBuster/workGenerators/BruteForceURLFuzz.java

```diff
@@ -154,7 +156,7 @@ private void makeList(int minLen, int maxLen, String baseCase, BaseCase baseCase
                     incrementCounter(x);
                     Thread.sleep(20);
                 } catch (InterruptedException ex) {
-                    Logger.getLogger(BruteForceURLFuzz.class.getName()).log(Level.SEVERE, null, ex);
+                    LOG.error("makeList " + ex.toString());
```

Comment:
I'd suggest debug, the thread being interrupted is not necessarily an issue/bug.